.github/workflows/run-e2e-on-fork.yml

```diff
@@ -0,0 +1,150 @@
+name: Run E2E on Fork
+
+permissions:
+  statuses: write # This is required for running set-status actions
+
+# This is to let only one instance of this workflow run against the same PR.
+concurrency:
+  group: run-e2e-${{ github.event.workflow_run.head_branch }}-${{ github.event.sender.login }} # This is to make the group name unique for a PR.
+  cancel-in-progress: true
+
+on:
+  workflow_run:
+    workflows: [Receive Fork Review]
+    types:
+      - completed
+
+jobs:
+  # check if approver has write/admin permissions
+  check-write-access:
+    runs-on: ubuntu-latest
+    # check if receive-fork-review workflow completed successfully
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: 'Get user permissions to check if it is either "write" or "admin"'
+        id: get-permissions
+        uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975
+        with:
+          result-encoding: string
+          script: |
+            const { data } = await github.rest.repos.getCollaboratorPermissionLevel({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              username: '${{ github.event.sender.login }}'
```

Comment:
Any reason we are not using `env` vars? (same question for all scripts bellow)

Reply:
Used env variables in latest commit